itests/org.jboss.tools.quarkus.integration.tests/src/org/jboss/tools/quarkus/integration/tests/project/AbstractNewQuarkusProjectTest.java

```diff
@@ -146,8 +154,35 @@ public static void checkProblemsView() {
 
 	@After
 	public void deleteProject() {
+		WorkbenchShellHandler.getInstance().closeAllNonWorbenchShells();
```

Comment:
You have this method with slight differences in three files, I would suggest some AbstractTest class a put it there and extend other classes with this.